Apps/Sandcastle/gallery/3D Tiles Hierarchy.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
```

Comment:
Minor, but can you please rename this to "3D Tiles Batch Table Hierarchy" or "3D Tiles Semantic Hierarchy", which ever you think is better aligned with the spec and what common 3D Tiles terminology will be.